airflow/sensors/base.py

```diff
@@ -220,7 +221,6 @@ def run_duration() -> float:
                 nonlocal started_at
                 return time.monotonic() - started_at
 
-        try_number = 1
```

Comment:
Have you tested it? I think there was an issue with this - for example logs in UI will be different for separate tries. cc @ashb with whom we discussed it some time ago due to #10790

Reply:
I had it running in a docker environment for a few days with no obvious issues (including some added debugging code to show the behavior of exponential_fallback was fixed as expected). Logs were all stored together properly in my environment.